docs/environment/performances.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Performances
+current_menu: performances
+introduction: Performance tuning and optimizations for serverless PHP applications on AWS Lambda.
+---
+
+This article sums up what to expect in terms of performances and how to optimize serverless PHP applications. The benchmarks included in this page [are published on GitHub](https://github.com/brefphp/benchmarks).
+
+## Overhead
+
+### HTTP
+
+The Bref [runtime for HTTP applications](/docs/runtimes/http.md) **does not add overhead to response times**.
+
+Here are execution times for an empty PHP application:
+
+|                  | 128M  | 512M | 1024M | 2048M |
+|------------------|------:|-----:|------:|------:|
+| Execution time   |  10ms |  1ms |   1ms |   1ms |
+
+Unless we use a particularly slow lambda (see the next section, 128M is not recommended), 1ms is the same execution time when PHP runs with Apache or Nginx on a classic server.
+
+We can see the same result with a "Hello world" written in Symfony (4ms being the minimum execution time of the framework):
+
+|                  | 128M  | 512M | 1024M | 2048M |
+|------------------|------:|-----:|------:|------:|
+| Execution time   |  58ms |  4ms |   4ms |   4ms |
+
+### Functions
+
+The Bref [runtime for PHP functions](/docs/runtimes/function.md) (non-HTTP applications) adds a small overhead:
+
+|                  | 128M  | 512M | 1024M | 2048M |
+|------------------|------:|-----:|------:|------:|
+| Execution time   | 175ms | 35ms |  16ms |  13ms |
+
+Since this runtime is often used in asynchronous scenarios (processing queue messages for example), it is often negligible.
+
+## CPU power and memory size
+
+The main factor that affects performances is a lambda's memory size. Indeed, [the amount of memory is proportional to the CPU power available](https://docs.aws.amazon.com/lambda/latest/dg/resource-model.html).
+
+In other words, **more memory means a more powerful CPU**. A 1024M lambda has a CPU 2 times more powerful than a 512M lambda.
+
+From 64M to 1,792M, applications run with one CPU (1,792M gives 1 full CPU). From 1,856M to 3,008M, applications run with 2 CPU. However, since PHP is single threaded and one lambda handles only 1 request at a time, using 2 CPU usually does not provide any benefit.
```

Comment:
This is not super clear to me. I know that 896M gives me 0.5CPU. When I first read this I intrepid it as "everything between 64 and 1792 gives to 1 CPU". 

Maybe write: 

>  From 64M to 1,792M, applications run with up to one CPU (1,792M gives 1 full CPU)

Reply:
:+1: thanks, I struggled with a clear explanation here and I prefer your version.